docs/what-is-nano/living-whitepaper.md

```diff
@@ -0,0 +1,40 @@
+Title: Living Whitepaper | Nano Documentation
+Description: Overview of the living whitepaper on the Nano protocol and node design
+
+# Nano - Digital money for the modern world
+
+--8<-- "wip-living-whitepaper.md"
+
+--8<-- "contributing-code.md"
+
+The original whitepaper for Nano was last updated in November 2017 and since then many improvements to the protocol have been made. The current node implementation has received updates every few months on average over 2018 and 2019.[^1] As these updates continue to make the network stronger over time, the static nature of a traditional whitepaper required too much effort to continually update and publish. To ensure information about Nano is kept as up-to-date as possible, a new "Living Whitepaper" is being managed through the existing documentation website, which is easier to update and is open source.[^2]
+
+## Protocol vs. Node
+
+The two main sections of the Living Whitepaper are the [Protocol Design](../protocol-design/introduction.md) and [Node Implementation](../node-implementation/introduction.md). Although they were structured to split the required elements to conform to the protocol (Protocol Design) away from optional improvements built into the current node (Node Implementation), there is some overlap between them. Where possible these overlaps have been highlighted; however, those interested in [contributing to the development of the protocol](https://docs.nano.org/node-implementation/contributing) or building another node implementation should analyze more closely the differences between these to ensure the necessary rules are followed.
```

Comment:
There are 3 links in this page that use the full docs.nano.org URL instead of a relative one:
- https://docs.nano.org/node-implementation/contributing
- https://docs.nano.org/glossary#protocol-version
- https://docs.nano.org/glossary#node-version

Reply:
Updated. Left the one in the references in footer as that will be helpful if this were to ever be printed to PDF.